app/views/director/counselors/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1> <%= @counselor.user.first_name %> <%= @counselor.user.last_name %> </h1>
+
+<h3> Alias: </h3> <%=@counselor.alias %>
+<h3> Training Status: </h3> <%=@counselor.training %>
+<h3> Account Status: </h3> <%=@counselor.account_status %>
+<h3> Email: </h3> <%=@counselor.user.email %>
+
+<br />
+
+<%= link_to "Edit Counselor", edit_counselor_path(:id => @counselor.id) %>
```

Comment:
Add a button to View All Counselors as well, please. We don't know 100% what page the user will be coming from when viewing this page, so giving them the choice to view all may be helpful to them. 😸